opal/util/Makefile.am

```diff
@@ -131,6 +131,9 @@ libopalutil_core_la_LIBADD = \
 libopalutil_core_la_DEPENDENCIES = \
         keyval/libopalutilkeyval.la
 
+# needed for fileno() (flex < 2.6.6)
+libopalutil_core_la_CFLAGS = -D_POSIX_C_SOURCE=200809L
```

Comment:
Can you put a comment in explaining what this is for?  Otherwise the rationale for this obscure flag will get lost in time...

Reply:
I added some details and the link to the github issue on flex.